core/src/main/java/hudson/model/ItemGroupMixIn.java

```diff
@@ -116,6 +116,8 @@ public boolean accept(File child) {
                 configurations.put(key.call(item), item);
             } catch (IOException e) {
                 Logger.getLogger(ItemGroupMixIn.class.getName()).log(Level.WARNING, "could not load " + subdir, e);
+            } catch (Exception e) {
+                Logger.getLogger(ItemGroupMixIn.class.getName()).log(Level.WARNING, "could not load " + subdir, e);
```

Comment:
If you are catching `Exception` anyway then you can delete the prior catch of `IOException` as redundant.